apps/snitch_core/test/data/schema/payment/card_payment_test.exs

```diff
@@ -53,7 +53,7 @@ defmodule Snitch.Data.Schema.CardPaymentTest do
       assert errors == [payment_id: {"does not refer a card payment", []}]
     end
 
-    test "with all good", context do
+    test "when added new card", context do
```

Comment:
This test reads like this: "CardPayment records when added new card" and this does not make sense. Please use a new describe block and improve the title as well -- it should explain the scenario unambigously.